### PR TITLE
Dialog UX improvements

### PR DESCRIPTION
**Pull request by trm217**
## Should be batch deployed with
- https://github.com/orbiting/styleguide/pull/456
- https://github.com/orbiting/republik-admin-frontend/pull/79
- https://github.com/orbiting/backends/pull/683

### CODE REVIEW

File: components/Discussion/DiscussionPreferences.js [hunk 1]
```diff
@@ -59,151 +64,301 @@ DiscussionPreferences.propTypes = {
...
+                  }}
+                >
+                  {t('components/DiscussionPreferences/commentAnonymously')}
+                </Checkbox>
+              </div>
             </div>
           )}
-          <Button onClick={this.onSave}>
+          <div {...styles.fieldWrapper}>
+            <Interaction.H2>
```
Comment (ovbm) [inline components/Discussion/DiscussionPreferences.js:287]: I'd reduce it to H3, it's a bit "in your face" right now

Comment (ovbm) [reply components/Discussion/DiscussionPreferences.js:287]: (same for annonymity h2)

File: components/Discussion/DiscussionPreferences.js [hunk 2]
```diff
@@ -59,151 +64,301 @@ DiscussionPreferences.propTypes = {
...
+                <Label>
+                  {t(
+                    'components/DiscussionPreferences/credentialAnonymityWarning'
+                  )}
+                </Label>
+              </div>
+            )}
+            {credentialSuggestions && (
+              <div {...styles.suggestedCredentialsWrapper}>
+                <Interaction.H3>
```
Comment (ovbm) [inline components/Discussion/DiscussionPreferences.js:314]: Use Label here, istead of H3

File: components/Discussion/DiscussionPreferences.js [hunk 3]
```diff
@@ -59,151 +64,301 @@ DiscussionPreferences.propTypes = {
...
+      <OverlayToolbar
+        title={t('components/DiscussionPreferences/modalTitle')}
+        onClose={onClose}
+      />
+      <OverlayBody noPadding>
+        <div
+          {...styles.previewWrapper}
+          {...colorScheme.set('borderBottomColor', 'divider')}
+        >
           <Interaction.P>
```
Comment (ovbm) [inline components/Discussion/DiscussionPreferences.js:239]: I'd remove the text here and put it into CommentHeaderProfile als a Labe
Screenshot 2021-11-26 at 15 07 43
l

File: components/Discussion/DiscussionPreferences.js [hunk 4]
```diff
@@ -59,151 +64,301 @@ DiscussionPreferences.propTypes = {
...
+                      href='#'
+                      onClick={() => setShowAllSuggestedCredentials(true)}
+                    >
+                      {t('components/DiscussionPreferences/showAllCredentials')}
+                    </A>
+                  )}
+              </div>
+            )}
+          </div>
+          <Button type='submit'>
```
Comment (ovbm) [inline components/Discussion/DiscussionPreferences.js:357]: Adding the `block`prop to the Button could look nice on mobile (optional)
Screenshot 2021-11-26 at 15 10 15

**Review by ovbm — APPROVED** [review]

**Review by ovbm — APPROVED** [review]

**Comment by trm217** [comment]
Will be merged together with https://github.com/orbiting/republik-frontend/pull/708

**Pull request closed without merging**